Library/Formula/gtk+.rb

```diff
@@ -29,10 +31,10 @@ def install
             "--prefix=#{prefix}",
             "--disable-glibtest",
             "--enable-introspection=yes",
+            "--with-gdktarget=quartz",
             "--disable-visibility"]
 
-    args << "--with-gdktarget=quartz" if build.without?("x11")
-    args << "--enable-quartz-relocation" if build.without?("x11")
+    args << "--enable-quartz-relocation" if build.with?("quartz-relocation")
```

Comment:
You don't need to wrap the `build.with?` here. `if build.with? "quartz-relocation"` is fine.